cert/lib/cert/commands_generator.rb

```diff
@@ -17,7 +17,7 @@ def run
       program :description, 'CLI for \'cert\' - Create new iOS code signing certificates'
       program :help, 'Author', 'Felix Krause <cert@krausefx.com>'
       program :help, 'Website', 'https://fastlane.tools'
-      program :help, 'GitHub', 'https://github.com/fastlane/fastlane/tree/master/cert#readme'
+      program :help, 'GitHub', 'https://docs.fastlane.tools/actions/cert/'
```

Comment:
"GitHub" is not correct here then...

Reply:
Talked to other contributors, this should probably be changed to "Documentation" on all cases where the Github link was replaced by a `docs.fastlane.tools` one.